apps/src/code-studio/components/progress/TeacherUnitOverview.tsx

```diff
@@ -198,7 +202,7 @@ interface TeacherUnitOverviewProps {
   // Define any props you need here
 }
 
-const initializeRedux = (
+export const initializeRedux = (
```

Comment:
Thoughts on this... initially I did this just to see if I could get the redux to work the way I wanted it to, and I did!

However, I was planning on making an `initializeRedux` function in `UnitCalendar` that would just focus on calendar data, but yesterday I thought I heard you say to keep it in here... perhaps because then we can get all the data we need for the calendar AND unit page in one fetch(?).  But I wanted to check in on this choice first.

Reply:
Yes, I think we can get all the info for unit and calendar in one fetch so it makes sense to have it all in the same call. Especially if we can have logic for the unit and calendar pages that only fetches data if it hasn't already been loaded.

Reply:
Optional improvement: 
Move this function to a separate utils file and rename it more specifically. Something like `setUnitSummaryReduxData`.

Reply:
I started to do this and it became a bit tangled... I do think it is a good idea, but would like to do it in it's own PR (or, at least, a smaller PR).  [Added a ticket to make sure we don't lose it though... ](https://codedotorg.atlassian.net/jira/software/projects/TEACH/boards/65?selectedIssue=TEACH-1423)